examples/aws-lambda-deployment/bal_build.out

```diff
@@ -0,0 +1,12 @@
+$ bal build
+Compiling source
+        wso2/aws_lambda_deployment:0.1.0
+
+Generating executable
+        @awslambda:Function: echo, ctxinfo, notifyS3
+
+        Run the following command to deploy each Ballerina AWS Lambda function:
+        aws lambda create-function --function-name $FUNCTION_NAME --zip-file fileb://<project-dir>/aws_lambda_deployment/target/bin/aws-ballerina-lambda-functions.zip --handler aws_lambda_deployment.$FUNCTION_NAME --runtime provided --role $LAMBDA_ROLE_ARN --layers arn:aws:lambda:$REGION_ID:134633749276:layer:ballerina-jre11:6 --memory-size 512 --timeout 10
+
+        Run the following command to re-deploy an updated Ballerina AWS Lambda function:
```

Comment:
```suggestion
        Run the following command to re-deploy an updated Ballerina AWS Lambda function:
```
Same as above.

Reply:
This is also fine as it is an instruction, which is being printed in the console output.